Makefile

```
@@ -62,7 +62,12 @@ endif
 # to ensure they are re-built (not considered up-to-date) when re-entering
 .DELETE_ON_ERROR:
 
-.PHONY: all modules clean help show always-update
+.PHONY: all modules check clean help show always-update
+
+check:
+	@sem --version >/dev/null && echo "ok - parallel installed"
```

Then an error would be reported like this:

/bin/sh: 1: sem: not found

```suggestion
.PHONY: all modules check clean help show always-update

check:
	@sem --version >/dev/null 2>&1 || echo "error - please install GNU parallel"
```

---

I have no strong opinion regarding the success message, so `ok - parallel installed` could be added for the success case if desired.